windows-gaming/driver/src/qemu.rs

```diff
@@ -46,6 +46,9 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path, clientpipe_path: &Path, monito
         debug!("Samba started");
     }
 
+    let ga = data.join("windows-gaming-ga.iso");
```

Comment:
naming: `gaIso` or something like that to easier see what this is. `ga` can have multiple meanings